SPECS/xapi.spec

```diff
@@ -63,25 +69,25 @@ Patch2: 0002-Xen-4.20-domctl_create_config.altp2m_ops.patch
 %endif
 
 # XCP-ng patches
+#   - Generated from our XAPI repository: https://github.com/xcp-ng/xen-api
+#   - git format-patch --no-numbered --no-signature  v25.6.0..v25.6.0-8.3
+#
 # Enables our additional sm drivers
-Patch1000: xapi-24.11.0-update-xapi-conf.XCP-ng.patch
-# Patch1001: in XCP-ng xs-clipboardd is named xcp-clipboardd
-Patch1001: xenopsd-22.20.0-use-xcp-clipboardd.XCP-ng.patch
-# Replace this if/when PR https://github.com/xapi-project/xen-api/pull/4188 is finalized
-Patch1002: xapi-24.39.0-open-openflow-port.XCP-ng.patch
-# Drop this patch when we don't want to support migration from older SDN controller anymore
-Patch1003: xapi-24.39.0-update-db-tunnel-protocol-from-other_config.XCP-ng.patch
-Patch1005: xapi-24.19.2-fix-ipv6-import.XCP-ng.patch
-# Fix fingerprints for CA certificates too
-Patch1007: xapi-24.19.2-more-fingerprint-field-updates-fixes.XCP-ng.patch
+Patch1001: 0001-xcp-ng-configure-xapi.conf-to-meet-our-needs.patch
+Patch1002: 0002-xcp-ng-renamed-xs-clipboardd-to-xcp-clipboardd.patch
 # Disable flaky test, see af3989235e62f5887a8cac08f4c1977726839297
-Patch1008: xen-api-24.39.1-test-disable-cancellable-sleep.patch
+Patch1003: 0003-xcp-ng-disable-cancellable-sleep.patch
 # #6280: traces to see observer not working on smapi component
-Patch1009: xen-api-24.39.1-debug-traces-for-is_component_enabled.patch
-# #6261: fix IPMI handling for hosts without IPMI (in 25.5.0)
-Patch1010: xen-api-24.39.1-0001-CA-399669-Do-not-exit-with-error-when-IPMI-readings-.patch
-Patch1011: xen-api-24.39.1-0002-rrdp-dcmi-remove-extraneous-I-argument-from-cli-call.patch
-Patch1012: xen-api-24.39.1-0003-CA-399669-Detect-a-reason-for-IPMI-readings-being-un.patch
+Patch1004: 0004-xcp-ng-add-debug-info-in-observer.patch
+Patch1005: 0005-xcp-ng-fix-IPv6-import.patch
+# check if https://github.com/xapi-project/xen-api/pull/4188 is fixed
+Patch1006: 0006-xcp-ng-open-close-openflow-port.patch
+# Drop this patch when we don't want to support migration from older SDN controller anymore
+Patch1007: 0007-xcp-ng-update-db-tunnel-protocol-from-other-config.patch
+# Below below this line we have cherry-pick:
+# Cherry-picked issues with metrics from 25.13
+Patch1008: 0008-CA-408126-rrd-Do-not-lose-ds_min-max-when-adding-to-.patch
+Patch1009: 0009-CA-408126-follow-up-Fix-negative-ds_min-and-RRD-valu.patch
 
```

Comment:
We will need to cherry-pick this one as well: https://github.com/xapi-project/xen-api/pull/6393
it is in 25.15

Reply:
This backport, as well as the other two, should be in a separate commit, not in the merge commit.

Arguably the rest of the patchqueue update could be in its own commit too:
* import commit
* merge commit, leaving TODOs
* patch update commit, only updating the existing patch queue
* add backports that are already known to be necessary

Reply:
@glehmann you moved all patch updates to a separate commit, which is fine, but above I was asking that we moved **new** backports to their own commit, to separate them from mere adaptation to the new sources.

Reply:
That's the only remaining comment. I still think added patches would better be in a separate commit, but I'm ready to approve as is.